dmriprep/workflows/dwi/util.py

```diff
@@ -0,0 +1,250 @@
+# -*- coding: utf-8 -*-
+
+"""
+Utility workflows
+^^^^^^^^^^^^^^^^^
+
+.. autofunction:: init_dwi_reference_wf
+.. autofunction:: init_enhance_and_skullstrip_dwi_wf
+
+"""
+
+from nipype.pipeline import engine as pe
+from nipype.interfaces import utility as niu, fsl, afni
+
+from niworkflows.engine.workflows import LiterateWorkflow as Workflow
+from niworkflows.interfaces.images import ValidateImage, MatchHeader
```

Comment:
```suggestion
from niworkflows.interfaces.images import ValidateImage
```

as per flake8, it seems MatchHeader is imported but not used.